### DISCUSSION TITLE
Squash Merge PR on `beta` does not trigger release

### DISCUSSION THREAD

**Discussion by severinlandolt** [semantic-release/semantic-release#2970] [Release workflow] [answered]
Hi everyone, Severin from @tremorlabs here.

Problem: After squash merging a PR on my `beta` branch, no release is triggered ([here's the PR](https://github.com/tremorlabs/tremor/pull/696))

- There was a `feat:` tag in the commit message
- My package.json has the correct branches for prerelease
```json
"release": {
    "branches": [
      "main",
      {
        "name": "beta",
        "prerelease": true
      },
      {
        "name": "beta-*",
        "prerelease": true
      }
    ],
    "plugins": [
      [
        "@semantic-release/commit-analyzer",
        {
          "preset": "conventionalcommits",
          "releaseRules": [
            {
              "type": "build",
              "release": "minor"
            }
          ]
        }
      ],
      "@semantic-release/npm",
      "@semantic-release/github"
    ]
  }
```

This might be relevant too:
https://github.com/tremorlabs/tremor/blob/beta/.github/workflows/release.yaml

After reading the [documentation again](https://semantic-release.gitbook.io/semantic-release/usage/configuration), I cannot tell what is missing for a beta release to happen.

Any help is appreciated!

**Comment by travi** [comment, marked as answer]
looks like a release was created here: https://github.com/tremorlabs/tremor/releases/tag/v3.8.0-beta.1. please provide more detail about what you are suggesting is missing

**Reply by travi** [reply]
`npm dist-tags ls @tremor/react` also shows that the current version published the the `beta` channel is `beta: 3.8.0-beta.1`

**Reply by severinlandolt** [reply]
Thanks! Did not catch that at all🤣 Thank you for maintaining such a useful piece of software
